server/routes/api/userRouter.js

```diff
@@ -5,6 +5,7 @@ const usersCtrl = require('../../controllers/api/userCtrl')
 
 router.post('/login', usersCtrl.logIn)
 router.post('/', usersCtrl.create)
+router.post('/signup', usersCtrl.create);
 // router.get('/check-token', ensuredLoggedIn, usersCtrl.checkToken)
 
```

Comment:
Feel free to delete router.post('/', usersCtrl.create)! No need for two separate paths to use the same controller function.